server/router/placeAndReviews.js

```diff
@@ -0,0 +1,32 @@
+const { getPlace, getPlaceReviews } = require('../models/queries/');
+
+module.exports = (req, res, next) => {
+  const { id, type } = req.query;
+  if (!id || !type) {
+    res.status(400).json({
+      message: 'place id and type required',
+    });
+  }
```

Comment:
throw an error and leave this for the error handling middleware (see my other comment)